apps/playground/src/app/components/modal/modal-error.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnDestroy, OnInit, inject } from '@angular/core';
+import { SkyModalBeforeCloseHandler, SkyModalInstance } from '@skyux/modals';
+
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-modal-error',
+  templateUrl: './modal-error.component.html',
+})
+export class ModalErrorComponent implements OnInit, OnDestroy {
+  public isError = false;
+
+  public errors: string[] | undefined;
+
+  #modalInstance = inject(SkyModalInstance);
+  #ngUnsubscribe = new Subject<void>();
+
+  public ngOnInit(): void {
+    this.#modalInstance.beforeClose
+      .pipe(takeUntil(this.#ngUnsubscribe))
+      .subscribe((handler) => this.#onClose(handler));
+  }
+
+  public ngOnDestroy(): void {
+    this.#ngUnsubscribe.next();
+    this.#ngUnsubscribe.complete();
+  }
+
+  public clearErrors(): void {
+    this.errors.pop();
+  }
+
+  public save(): void {
+    this.#modalInstance.save();
+  }
+
+  public cancel(): void {
+    this.#modalInstance.cancel();
+  }
+
+  #onClose(handler: SkyModalBeforeCloseHandler): void {
+    if (this.isError && handler.closeArgs.reason !== 'cancel') {
+      this.errors = ['Sample error', 'Sample error 2'];
```

Comment:
Can we put a multiline error in here so we can see what it looks like in the playground?

Reply:
I would also create a new modal visual test that includes 2-3 error messages (one modal), with one of the errors being multiline so we can catch any spacing changes in the future.

Reply:
added